rest_framework_docs/static/rest_framework_docs/js/components/helpers/input.js

```diff
@@ -2,6 +2,11 @@ var React = require('react');
 
 var Input = React.createClass({
 
+  removeField: function (fieldName, event) {
+    event.preventDefault();
```

Comment:
It is placed within a `<label>` so on Click it focuses to the input which is about to get removed.